app/page.tsx

```diff
@@ -15,7 +15,7 @@ export default function Page() {
             <a href="https://nextjs.org/learn/" className="text-blue-500">
               Next.js Learn Course
             </a>
-            , brought to you by Vercel.
+            , brought to you by myself.
```

Comment:
The name should start with capital letter

Reply:
ok noted

Reply:
Okay resolving it. Thank you!